client/src/app/site/pages/meetings/pages/participants/modules/groups/components/group-list/group-list.component.ts

```diff
@@ -277,4 +284,11 @@ export class GroupListComponent extends BaseMeetingComponent implements OnInit,
         }
         return true;
     }
+
+    /**
+     * Function to allow to edit the external_id
+     */
+    public get allowExternalId(): boolean {
+        return this.operator.isMeetingAdmin || this.operator.isAccountAdmin;
+    }
```

Comment:
Account admins are not allowed to update the external ID, please remove them

Reply:
Are orga admins or superadmins allowed to update the external Id?